src/assessment.handlebars

```diff
@@ -622,13 +622,403 @@
+        </ul>
+      <h2>Unhappy with the Decision?</h2>
+      <h3>First Steps</h3>
+      <p>You may be unhappy with the decision that you get from the DWP.</p>
+      <p>The DWP say that they will try and contact you to explain their decision once you have got the letter saying that you aren’t eligible for PIP.</p>
+      <p>When they do, you can tell them if you don’t agree with the decision. You can also give them more information to support what you are saying, for example if your circumstances change.</p>
+      <p>It is possible, though unlikely, that the DWP will decide to award you the benefit at this point.</p>
+      <h3>Mandatory Reconsideration</h3>
+      <p>More likely, your next step is to request what is called a 'mandatory reconsideration'.</p>
+      <p>This means formally asking the DWP to look again at their decision. The quickest way to do this is by phoning the DWP. If you do not want to call them you must write to the address on your decision notice. You need to explain why you do not agree with the decision, you can also include further supporting information.</p>
+      <p>If you are unhappy with the decision that you have received, you need to take action quite quickly. You only have one month from the date of the decision letter to ask for a mandatory reconsideration.</p>
+      <p>The DWP will then review its decision, and will write to you to tell you what that decision is, and to explain why they have reached it.</p>
+      <h3>Appeal</h3>
+      <p>If you are still unhappy with the decision you have received, you can appeal against it.</p>
+      <p>Again, you need to act quickly. You must make your appeal in writing or on form SSCS1 within one month of the date of the letter telling you the result of your mandatory reconsideration. You must send a copy of the DWP’s mandatory reconsideration notice with your appeal.</p>
+      <p>We recommend that you get support from a CAB or other advice agency with submitting the mandatory reconsideration. We even more strongly recommend that you seek such support if you are appealing against a decision.</p>
+      <p>Information about where to get help is available from <a href="https://www.citizensadvice.org.uk/about-us/how-we-provide-advice/advice/" target="_blank">Citizens Advice</a> and <a href="http://www.adviceuk.org.uk/looking-for-advice-2/" target="_blank">Advice UK</a>.</p>
+      <ul aria-label="menu">
+        <li>
+          <button class="nav-link pick" data-action="start-or-resume">Try some questions</button>
+        </li>
+        <li>
+          <button class="nav-link break" data-action="break">Have a break</button>
+        </li>
+        <li>
+          <button class="nav-link break" data-action="about-pip">Read more about PIP</button>
+        </li>
```

Comment:
Change this one to 'Read less about PIP' ... @harryharrold copy for button getting back to the Essential guide from the html'd pdf.

See http://esa-assessment.support/#guide for the live example of same thing on ESA which we also need to change.